analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpEquivalenceChecker.cs

```diff
@@ -36,6 +36,33 @@ internal static class CSharpEquivalenceChecker
         public static bool AreEquivalent(SyntaxList<SyntaxNode> nodeList1, SyntaxList<SyntaxNode> nodeList2) =>
             Common.EquivalenceChecker.AreEquivalent(nodeList1, nodeList2, NodeComparator);
 
+        public static bool AreEquivalentWithEqualInvocations(SyntaxList<SyntaxNode> nodeList1, SyntaxList<SyntaxNode> nodeList2, SemanticModel model)
```

Comment:
left vs right

Reply:
It's not really left and right in this case.
It can be any two groups of statements. 
I changed it for `first` and `second` also here.